src/connector/src/source/iceberg/mod.rs

```diff
@@ -129,6 +129,10 @@ impl SplitMetaData for IcebergSplit {
     fn update_with_offset(&mut self, _start_offset: String) -> ConnectorResult<()> {
         unimplemented!()
     }
+
+    fn get_encoded_offset(&self) -> String {
+        unimplemented!()
```

Comment:
Is this okay?

Reply:
modify to `unreachable()`, since iceberg source cannot be used as stream source.